core/arch/arm/kernel/virtualization.c

```diff
@@ -138,6 +140,12 @@ void virt_init_memory(struct tee_mmap_region *memory_map)
 {
 	struct tee_mmap_region *map;
 
+	/*
+	 * We can't use VCORE_UNPG_RW_SZ value directly, because it is altered
+	 * by ASLR.
+	 */
+	tee_ram_size = VCORE_UNPG_RW_SZ - boot_mmu_config.load_offset;
```

Comment:
I guess that means that you haven't found a way either to avoid having `VCORE_*_SZ` altered by ASLR.